Code/GraphMol/MolAlign/O3AAlignMolecules.cpp

```diff
@@ -1221,8 +1220,8 @@ O3A::O3A(int (*costFunc)(const unsigned int, const unsigned int, double,
          double (*weightFunc)(const unsigned int, const unsigned int, void *),
          double (*scoringFunc)(const unsigned int, const unsigned int, void *),
          void *data, ROMol &prbMol, const ROMol &refMol, const int prbCid,
-         const int refCid, boost::dynamic_bitset<> *prbHvyAtoms,
-         boost::dynamic_bitset<> *refHvyAtoms, const bool reflect,
+         const int refCid, const boost::dynamic_bitset<> &prbHvyAtoms,
```

Comment:
Switching these two `dynamic_bitset` arguments to references ends up clearing up a bug (likely never observed in real code) when `prbHvyAtoms` was null.